src/docs/software/using/python.md

```diff
@@ -0,0 +1,116 @@
+Python is an interpreted, object-oriented, high-level programming language with dynamic semantics.
+Its high-level built in data structures, combined with dynamic typing and dynamic binding, 
+make it very attractive for Rapid Application Development, as well as for use as a scripting or 
+glue language to connect existing components together. Python's simple, easy to learn syntax 
+emphasizes readability and therefore reduces the cost of program maintenance. 
+
+
+## Python versions
+Sherlock features two different versions of Python: 2.7 and 3.6.1. 
```

Comment:
For the sake of simplicity, I'd put only major.minor version, so `2.7 and 3.6`
But since users will have to use the full major.minor.patch version when they load the modules, I'm fine with `2.7.13 and 3.6.1` as well. As long as the two version numbers are consistent. :)